MoviesMVP/MoviesMVPTests/MockRouter.swift

```diff
@@ -0,0 +1,32 @@
+// MockRouter.swift
+// Copyright © RoadMap. All rights reserved.
+
+@testable import MoviesMVP
+import XCTest
+
+/// Мок роутера
+final class MockRouter: RouterProtocol {
+    // MARK: - Public property
+
+    public var isDetailShowed: Bool = false
```

Comment:
тут паблик нужен? Если нет, убери
```suggestion
    var isDetailShowed: Bool = false
```